Description: Disable tests which work fine with local X but hang under xvfb-run
Author: Axel Beckert <abe@debian.org>
Last-Update: 2014-10-10

--- a/testsuite/Makefile.am
+++ b/testsuite/Makefile.am
@@ -18,7 +18,6 @@
   test_bug_2876372.pro \
   test_bug_2892631.pro \
   test_bug_2949487.pro \
-  test_bug_2974380.pro \
   test_bug_3033108.pro \
   test_bug_3054361.pro \
   test_bug_3055720.pro \
@@ -43,7 +42,6 @@
   test_bug_3152899.pro \
   test_bug_3189072.pro \
   test_bug_3199465.pro \
-  test_bug_3275334.pro \
   test_bug_3285659.pro \
   test_bug_3286031.pro \
   test_bug_3288652.pro \
@@ -74,16 +72,12 @@
   test_ce.pro \
   test_clip.pro \
   test_common.pro \
-  test_congrid.pro \
   test_constants.pro \
   test_convert_coord.pro \
   test_correlate.pro \
   test_deriv.pro \
-  test_device.pro \
   test_dicom.pro \
   test_erfinv.pro \
-  test_execute.pro \
-  test_extra_keywords.pro \
   test_fft.pro \
   test_fft_dim.pro \
   test_fft_leak.pro \
@@ -122,7 +116,6 @@
   test_memory.pro \
   test_message.pro \
   test_modulo.pro \
-  test_moment.pro \
   test_mpfit.pro \
   test_multiroots.pro \
   test_nans_in_sort_and_median.pro \
@@ -130,10 +123,8 @@
   test_netcdf.pro \
   test_null.pro \
   test_obj_new.pro \
-  test_plot_oo.pro \
   test_plotting_ranges.pro \
   test_pmulti.pro \
-  test_pmulti_basic.pro \
   test_postscript.pro \
   test_product.pro \
   test_ps_decomposed.pro \
@@ -144,7 +135,6 @@
   test_python_module_2.pro \
   test_qromb.pro \
   test_qromo.pro \
-  test_random.pro \
   test_readf.pro \
   test_reads.pro \
   test_rebin.pro \
@@ -167,13 +157,11 @@
   test_suite.pro \
   test_systime.pro \
   test_trisol.pro \
-  test_tv.pro \
   test_typename.pro \
   test_url.pro \
   test_voigt.pro \
   test_wait.pro \
   test_wavelet.pro \
-  test_window_background.pro \
   test_where.pro \
   test_zeropoly.pro \
   test_zip.pro \
